keyboards/handwired/2x5keypad/2x5keypad.c

```diff
@@ -0,0 +1,8 @@
+#include "2x5keypad.h"
+
+void keyboard_pre_init_user(void)
```

Comment:
```suggestion
void matrix_init_kb(void)
```

Reply:
Ok, I don't understand this suggestion.  Should I replace keyboard_pre_init_user() with matrix_init_kb() or should my keyboard_pre_init_user() call matrix_init_kb() in its body?

Reply:
`keyboard_pre_init_user()` should be renamed to `matrix_init_kb()` and call `matrix_init_user()` before setting up the LEDs. The core implementation of this function simply calls `matrix_init_user()`, so we need to do the same thing in the overriding function so that it is still called for keymaps.